astropy/coordinates/baseframe.py

```diff
@@ -637,14 +641,16 @@ def _infer_repr_info(cls, repr_info):
         return repr_info
 
     @classmethod
-    def _create_readonly_property(cls, attr_name, value):
+    def _create_readonly_property(cls, attr_name, value, docstring=None):
         private_attr = '_' + attr_name
 
         def getter(self):
             return getattr(self, private_attr)
 
         setattr(cls, private_attr, value)
         setattr(cls, attr_name, property(getter))
```

Comment:
does setting the docstring via the property not work?

```suggestion
        setattr(cls, attr_name, property(getter, doc=docstring))
```

Reply:
\<head smack\>  Yes, that's the better way to do it.  Changed.